.env.sample

```diff
@@ -91,3 +91,4 @@ LOG_LEVEL = info
 
 REDIS_HOST=
 REDIS_PORT=
+REDIS_PASSWORD=
```

Comment:
REDIS_PASSWORD is required for using REDIS Cloud. I've also updated the install.md file